x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -934,9 +934,10 @@ export function calculateDelay(attempts: number) {
   if (attempts === 1) {
     return 30 * 1000; // 30s
   } else {
-    // get multiples of 5 min
     const defaultBackoffPerFailure = 5 * 60 * 1000;
-    return defaultBackoffPerFailure * Math.pow(2, attempts - 2);
```

Comment:
Any reason why we have to keep this special case for `attempts === 1`? Seems like we could just use `random(Math.min(maxDelay, defaultBackoffPerFailure * Math.pow(2, attempts)))` in all cases and then we can reference the [backoff algorithm](https://aws.github.io/aws-iot-device-sdk-embedded-C/202012.00/libraries/standard/backoffAlgorithm/docs/doxygen/output/html/index.html) in a comment.

Reply:
I left it in bc it was asked for in original implementation, but I can remove it you think that's best.

Reply:
Gotcha. We can leave that in then, with a comment maybe? And I don't think we need to adjust the `attempts` value in the `Math.pow`, WDYT?

Reply:
Sure I can add a comment, I can also remove it if you want me to I don't feel strongly about it. I adjusted the attempts to match what the original issue was asking, if you don't adjust the attempts than the 2nd attempt would retry between 0 - 20m instead of 0 - 5m. If we remove the `attempts === 1` condition, I still think we would need to adjust the attempts to `attempts - 1`.

Reply:
I see. Yea I think a comment would be useful to know what we're trying to achieve :)

Reply:
Yes I agree, thanks!

Reply:
Comment updated in this commit https://github.com/elastic/kibana/pull/173779/commits/3f6afe6d6881e99df1350d739f2cdbbfae71fb0f